airbyte-integrations/connectors/source-greenhouse/source_greenhouse/manifest.yaml

```diff
@@ -125,13 +127,14 @@ definitions:
       requester:
         $ref: "#/definitions/requester"
         path: "applications/{{ stream_slice.parent_id }}/demographics/answers"
-      partition_router:
-        class_name: source_greenhouse.components.GreenHouseSubstreamSlicer
-        parent_stream: "#/definitions/applications_stream"
-        request_cursor_field: "updated_after"
-        stream_slice_field: "parent_id"
-        cursor_field: "updated_at"
-        parent_key: "id"
+    incremental_sync:
```

Comment:
after inspecting this connector a bit, this really should be an incremental_sync custom component instead of a custom partition_router because it has a `cursor_field`. In the existing code, we would inadvertently only perform full refreshes since we didn't seen `incremental_sync` component and therefore assumed it was only full refresh.

This also is more in line w/ our ideal structure where `incremental_sync` dictates the sync capabilities.